HarmonyHub/Entities/Activity.cs

```diff
@@ -19,7 +20,7 @@ public class Activity : IComparable<Activity>
         [DataMember(Name = "activityTypeDisplayName")]
         public string ActivityTypeDisplayName { get; set; }
         [DataMember(Name = "controlGroup")]
-        public IList<object> ControlGroup { get; set; }
+        public IList<object> ControlGroups { get; set; }
```

Comment:
Oops, missed that one. Was focused on the devices and forgot to check the activities.